docs/modules/ROOT/pages/explanations/decisions/capacity-alerting.adoc

```diff
@@ -0,0 +1,108 @@
+= Capacity Alerting and Incident Prevention
```

Comment:
```suggestion
= Handling of Capacity Alerts for Incident Prevention
```

or 

```suggestion
= Alert Service User on Service Instance Capacity
```

From my understanding of the problem description, one of these suggestions would describe it much better?